app/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.hiya.jacoco-android'
+apply plugin: 'com.akaita.android.easylauncher'
```

Comment:
Does this work with other launchers?

Reply:
Yes, I've tasted this on Samsung's One UI too.

Reply:
@EmmanuelMess For your reference:
![image](https://user-images.githubusercontent.com/55531939/119286825-d74e9380-bc62-11eb-93dd-e8daf1dce630.png)
![image](https://user-images.githubusercontent.com/55531939/119286837-de75a180-bc62-11eb-874d-0e5e095830ac.png)

Reply:
Works on KISS too.